src/remark/models/slideshow.js

```diff
@@ -30,14 +30,17 @@ function Slideshow (events, dom, options, callback) {
   self.getSlideByName = getSlideByName;
   self.getSlidesByNumber = getSlidesByNumber;
 
-  self.togglePresenterMode = togglePresenterMode;
   self.toggleHelp = toggleHelp;
-  self.toggleBlackout = toggleBlackout;
-  self.toggleMirrored = toggleMirrored;
-  self.toggleFullscreen = toggleFullscreen;
-  self.createClone = createClone;
 
-  self.resetTimer = resetTimer;
+  if (!options.disablePresentationShortcuts) {
+    self.togglePresenterMode = togglePresenterMode;
+    self.toggleBlackout = toggleBlackout;
+    self.toggleMirrored = toggleMirrored;
+    self.toggleFullscreen = toggleFullscreen;
+    self.createClone = createClone;
+
+    self.resetTimer = resetTimer;
+  }
```

Comment:
Double negative statement "not disabled" ... opportunity to refactor the property to "enable presentation shortcuts"  as default, and then "disable" becomes `enablePresentationShortcuts: false`